OrbitGl/SamplingReport.cpp

```diff
@@ -134,7 +134,7 @@ void SamplingReport::OnCallstackIndexChanged(size_t index) {
   if (index < selected_sorted_callstack_report_->callstacks_count.size()) {
     const CallstackCount& cs = selected_sorted_callstack_report_->callstacks_count[index];
     selected_callstack_index_ = index;
-    callstack_data_view_->SetCallStack(callstack_data_->GetCallStack(cs.callstack_id));
+    callstack_data_view_->SetCallStack(*callstack_data_->GetCallStack(cs.callstack_id));
```

Comment:
This was changed and reverted again, is this intended?

Reply:
Yes, I'll squash these 2 commits, so it shouldn't affect the history (with bare pointer I think it make sense to have `const CallStack* callstack` in the SetCallStack, but as we create a copy I would prefer to pass a const ref as an argument).